.github/pull_request_template.md

```diff
@@ -11,4 +11,9 @@ If these changes modify code paths involving cryptography, the opening of files
 
  - [ ] I have tested these changes in the appropriate Qubes environment
  - [ ] I do not have an appropriate Qubes OS workstation set up (the reviewer will need to test these changes)
- - [ ] These changes should not need testing in Qubes
\ No newline at end of file
+ - [ ] These changes should not need testing in Qubes
+
+ If the client interacts with a new file, or no longer interacts with a file, the AppArmor profile may need to be updated. Please check as applicable:
```

Comment:
Perhaps we could generalize this a bit more. Packaging changes are also required if the client code includes a new (non-python-code) file. Perhaps we can generalize as follows
`the AppArmor profile may need to be updated. Please check as applicable:` -> `packaging logic (e.g. AppArmor profile) may need to be updated. Please check as applicable:`

` I have submitted a separate PR for the AppArmor profile update to the [packaging repo](https://github.com/freedomofpress/securedrop-debian-packaging) ` ->   `I have submitted a separate PR for the packaging changes or AppArmor profile to the [packaging repo](https://github.com/freedomofpress/securedrop-debian-packaging) `

`No AppArmor profile update is required for these changes `->` No packaging changes are required for these changes `

Reply:
Tried to address this in a2ce6564a5781d62f71d4f21a020123a363ccaf7 and also added an "I don't know" option; thoughts?